pkg/repository/repository.go

```diff
@@ -89,8 +94,13 @@ type gitStatusInterface interface {
 }
 
 func NewRepository(path string) Repository {
+	url, err := giturls.Parse(path)
+	if err != nil {
+		oktetoLog.Infof("could not parse url: %w", err)
```

Comment:
Why do we use Info level for errors?

Reply:
`oktetoLog.Infof` logs the error into the log file. If we use another level for the errors, we'll show the log to the user.

Reply:
Understood! Thanks @jLopezbarb